.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+/** @module {CjsModule} */
```

Comment:
what is the purpose of this jsdoc comment? Every `.js` file in a node module is a CJS module.

Reply:
I originally started doing this because I wanted to omit file extensions from my config files. This was before the import assertions proposal made any progress. Still, I noticed how convenient it was to quickly identify the context of my JS module code w/o having to think about Node's resolution algorithms. I will get rid of it.